src/modules/AutoinstPartPlan.rb

```diff
@@ -340,7 +340,12 @@ def ReadHelper
       _StorageMap = Builtins.eval(Storage.GetTargetMap)
 
       _StorageMap = _StorageMap.select do |d, p|
-        ok = d != "/dev/evms" && d != "/dev/nfs"
+        ok = d != "/dev/evms"
+        if( ok && d == "/dev/nfs" && p["partitions"] != nil)
+          # Checking if the nfs partition has a root partition.
```

Comment:
The terminlogy is a bit strange here: There is no "nfs partition", "/dev/nfs" is a container for nfs volumes, that is nfs mounts.